webpack/assets/javascripts/react_app/components/Layout/Navigation.js

```diff
@@ -82,11 +82,23 @@ const Navigation = ({
     [items.length, currentPath]
   );
 
+  const [currentExpandedSecondery, setCurrentExpandedSecondery] = useState(
+    null
+  );
   const [currentExpanded, setCurrentExpanded] = useState(
     subItemToItemMap[pathFragment(getCurrentPath())]
   );
   useEffect(() => {
     setCurrentExpanded(subItemToItemMap[pathFragment(getCurrentPath())]);
+    groupedItems.forEach(({ groups }) => {
+      groups.forEach(({ groupItems, title }) => {
+        groupItems.forEach(({ href }) => {
+          if (href === pathFragment(getCurrentPath())) {
+            setCurrentExpandedSecondery(title);
```

Comment:
Maybe `groupItems.some` could be used instead of this cycle?

Reply:
yeah, fixed